pkgs/tools/security/hashcat/default.nix

```diff
@@ -28,7 +28,9 @@ stdenv.mkDerivation rec {
 
   preFixup = ''
     for f in $out/share/hashcat/OpenCL/*.cl; do
+      # Rewrite files to be included for compilation at runtime for opencl offload
       sed "s|#include \"\(.*\)\"|#include \"$out/share/hashcat/OpenCL/\1\"|g" -i "$f"
+      sed "s|#define COMPARE_\([SM]\) \"\(.*\.cl\)\"|#define COMPARE_\1 \"$out/share/hashcat/OpenCL/\2\"|g" -i "$f"
```

Comment:
A patch would be easier to review and maintain I think.

Reply:
There is quite a lot of files involved, looking at the source, I'm not sure a patch is any easier. That would carry a whole lot more noise than a one line sed.

See:
https://github.com/hashcat/hashcat/tree/master/OpenCL
https://github.com/hashcat/hashcat/search?q=COMPARE_S